core/geometry/src/solid/RotationalSweep.ts

```diff
@@ -41,14 +41,15 @@ export class RotationalSweep extends SolidPrimitive {
     super(capped);
     this._contour = contour;
     this._normalizedAxis = normalizedAxis;
-    this.capped = capped;
     this._sweepAngle = sweepAngle;
+    this.capped = capped;
```

Comment:
this._capped will already have been set by the call to super...

Reply:
correct. I will remove this line